src/ui/ui.cpp

```diff
@@ -97,6 +97,7 @@ void UI::load_loop(int loop_num) {
     int id = db_manager.find_loop_id(std::to_string(loop_num));
     std::vector<std::string> track_paths = db_manager.load_tracks_by_loop(id);
 
+    if (track_paths[0] == "") std::cout << "No tracks with path for that loop_id" << std::endl;
     for ( unsigned long i=0; i<track_paths.size(); i++)

```

Comment:
I meant to ask, how large will we expect track_paths to be? Do we need it to be long to handle the size? or will a standard int suffice?

Reply:
Track paths will be a list of strings that contain the paths for each track associated with the loop; I guess at most there will be 4

Reply:
We can probably change this to an int then. We won't be going over the size limit of an int